modules/uploads/utils/env.ts

```diff
@@ -0,0 +1,38 @@
+import { Config } from "../config.ts";
+
+export interface S3EnvConfig {
+	UPLOADS_AWS_ACCESS_KEY_ID: string;
+	UPLOADS_AWS_SECRET_ACCESS_KEY: string;
+}
+
+export interface S3Config {
+    bucket: string;
+    region: string;
+    endpoint: string;
+
+    accessKeyId: string;
+    secretAccessKey: string;
+}
+
+export function getS3EnvConfig(s3Cfg: Config["s3"]): S3Config | null {
+    const {
+        UPLOADS_AWS_ACCESS_KEY_ID,
+        UPLOADS_AWS_SECRET_ACCESS_KEY,
+    } = Deno.env.toObject() as Partial<S3EnvConfig>;
+
+    if (
+        !UPLOADS_AWS_ACCESS_KEY_ID ||
+        !UPLOADS_AWS_SECRET_ACCESS_KEY
+    ) {
+        return null;
+    }
```

Comment:
Should allow overriding default env var that holds the secret like we do in SendGrid